### DISCUSSION TITLE
Trust Spanning Protocol - Terms & Concepts

### DISCUSSION THREAD

**Discussion by neiljthomson** [trustoverip/trust-spanning-protocol#29] [General]
**Purpose** - to capture and define (or refine) new terms and concepts that apply to the ToIP Technical Architecture, components in the architecture, etc.

**Format:**

 - **Term** - single discussion post - add new terms for which a definition does not exist
   - reply to the post to add a new term
 - **Term + Definition** - a single discussion topic for each term and a draft definition
   - reply to provide comments, proposed changes, etc. to the definition or to add additional description material (e.g., links to reference)

**Comment by neiljthomson** [comment]
## New Terms

- add a new term by replying to _this_ post

**Reply by Oskar-van-Deventer** [reply]
TL;DR Proper terminology design is different from collecting lists of terms and definitions.

Hi Neil @neiljthomson ,

Whereas proper terminology design is be essential for any technical specification, and especially in the ToIP domain, randomly posting terms and definitions will likely be counter-productive, and potentially leading to inane semantic discussions. 

The [ToIP CTWG group](https://github.com/trustoverip/concepts-and-terminology-wg), led by Drummond Reed (@talltree) and Rieks Joosten (@RieksJ) has developed an effective method for terminology design. This method starts with discussion which concepts are needed and their context. Next step is achieving consensus on what are the criteria to distinguish a particular concept from something that is not that concept. Lack of consensus often indicates that there is more than one concept at play. Only once there is a common understanding about the criteria that defines the concept, the group -in consensus- agrees on a term that best fits the concept.

Moreover, once a group has committed to a concept in its context, its defining criteria, and its term, the group should be vigilant against any use of the agreed term for a different concept/meaning. We don't want to get W3C-CCG scenes at ToIP ...

Oskar

**Reply by neiljthomson** [reply]
Hi Oskar @Oskar-van-Deventer,

In yesterday's **ToIP Technology Stack Working Group Plenary** meeting, I proposed (and it was accepted) that - in support of the more formal process of defining terms and concepts - that a **Discussion Issue** to collect **candidate** Terms & Concepts and **candidate** Definitions in the following GitHub Repositories:

 - Trust Spanning Protocol (here) 
 - [Trust Registry TF](https://github.com/trustoverip/tswg-trust-registry-tf)
 - [Tech Arch](https://github.com/trustoverip/TechArch)

These candidates will be discussed and rolled into the formal Terms Wikis.

**Comment by talltree** [comment]
Noting that there is a discussion about the term `authentic data` in [this discussion thread](https://github.com/trustoverip/trust-spanning-protocol/discussions/22#discussioncomment-5296092) (which culminated in the suggestion we need to qualify the term (e.g., `ToIP authentic data`) or use another term (e.g., `verifiable data`).

**Reply by RieksJ** [reply]
I suggest that people that want to contribute bringing this discussion to a close run the following experiment:
- in any text that they write, they do NOT use the term `authentic data`, but instead write a criterion that has the meaning they want to convey. Under the assumption that they otherwise would have used the term multiple times, they would now thus write that criterion multiple times. 
- when the text is done, they review the sentences with the criterion, to see if they all make sense. If not, the criterion should be changed (meaning that the text changes at the various locations as well), until all sentences make sense. If this does not happen, it is likely that you're talking about multiple concepts (because you end up with multiple, differing criteria).
- when done, for all criteria you come up with a name (it needn't be, but might be `authentic data`; obviously, if you had multiple criteria, you need to come up with different names as well), and you propose as many definitions as you have pairs (name, criterion).
- then you wait to see how others will react...

**Reply by mwherman2000** [reply]
> `ToIP authentic data`

1. Need to property define and differentiate between:
   - Authentic Conversations
   - Authentic Messages
   - Authentic Attachments
   - ...and only then, Authentic Data
2. It doesn't make sense to me (and likely most English language folks) to have a _ToIP_ specific definition of Authentic Data that is different/in conflict with a conventional English understanding of the term

**Reply by neiljthomson** [reply]
@mwherman2000 - we have a number of proposed definitions of crypto- and governance-based traceability to the source of data. The "phrased used" is "Authentic Data". "Authentic Messages" have been defined as messages which are "tamper evident" and can be confirmed/proven as being provided by the "message sender". Feel free to specify other "terms" or "phrases" and the definitions of "the criterion".

PS: the problem with English words, is they have several meanings, some of which do not apply to "data". For example, the Oxford English Dictionary defines Authentic to include "true, accurate," where Webster's Dictionary only comes as close as "worthy of acceptance or belief as conforming to or based on fact". 

Discussions around (electronic) Data points to "Veracity" of data as a needed and separate term from "tramper evident" and "provably provided by the message sender."

**Reply by RieksJ** [reply]
This is missing the point altogether, at least from my perspective. The question I would like to see each of you answer before we do anything else, is this: "what criteria do you use to determine what is, and what is not, (whatever it is you call)  'authentic data'?" The answer to this question should enable me to make the same distinctions as you do regarding that concept. So if anyone of us things of something that exists (an object, bits, an idea, whatever), I should be able to evaluate that criterion on this something, which would result in a yes (the something fulfills the criteria) or no (the something does not fulfill the criteria).

**Reply by mwherman2000** [reply]
> 1. Need to property define and differentiate between:
>    * Authentic Conversations
>    * Authentic Messages
>    * Authentic Attachments
>    * ...and only then, Authentic Data

The sole purpose for listing these 4 terms in this order is lead to the conclusion that Authentic Data is not actually needed for defining a Trust Spanning Layer Protocol and any attempt to do so will be extremely problematic.

In the every day world (aka reality), Authentic Data can only be construed as an _unconscious contraction_ of the term _Authentic Line-of-Business (LOB) Data_ ...of which no general proof mechanism is available. 

Reference: https://hyperonomy.com/2021/02/01/ssi-unconscious-contractions/

**Comment by neiljthomson** [comment]
@RieksJ I would suggest that there is no agreed, simple, singular definition of "authentic data" criteria that would be easy to use in documentation in all contexts. 

I would suggest an example of "criteria" that has a clear "criteria phrase" that is simple and generally agreed to, vs Authentic Data.

**Reply by mwherman2000** [reply]
Agreed (I think). See https://github.com/trustoverip/trust-spanning-protocol/discussions/29#discussioncomment-5516081

**Reply by RieksJ** [reply]
I never said that there should be one (single) singular definition of "authentic data". In fact, I would guess the problems with properly defining it originate from having multiple, different, yet relevant concepts that some refer to as "authentic data" where others do not. Rather than arguing over the term, my proposal of creating criteria aims to help discover what these (other) relevant concepts are. If you come up with criteria for the concepts you think are relevant, it only means that I (and perhaps some others) get to understand you, and your way of thinking perhaps. That's what is needed prior to anything else.

**Comment by Oskar-van-Deventer** [comment]
Please see Thread #41, in which we invite everyone to help select the most urgent concepts/terms that require a more precise definition.
